Playground/CMakeLists.txt

```diff
@@ -1,20 +1,17 @@
 set(exe_name Playground)
-########################################
+###############################################
 # External Libraries
 xpGetExtern(externIncs externLibs PUBLIC boost)
```

Comment:
You don't need to do xpGetExtern any more. I will put a comment later down how you do it